README.md

```diff
@@ -42,14 +42,17 @@ kubectl apply -f https://raw.githubusercontent.com/llamastack/llama-stack-k8s-op
 
 ### Deploying the Llama Stack Server
 
-1. Deploy Inference provider server (ollama, vllm etc)
-2. Create LlamaStackDistribution CR to get the server running. Example-
+1. Deploy the inference provider server (ollama, vllm etc). Example to deploy a new ollama server:
+```
+bash hack/deploy-ollama.sh
+```
+2. Create LlamaStackDistribution CR to get the server running. Example:
 ```
 apiVersion: llamastack.io/v1alpha1
 kind: LlamaStackDistribution
 metadata:
   name: llamastackdistribution-sample
-  namespace: <your-namespace>
+  namespace: ollama-dist
```

Comment:
We do not require the CR to be created in same namespace as inference provider namespace

Reply:
I think its better to omit the value